### PR TITLE
Twin on device

### PR DESCRIPTION
**Pull request by mamokarz**
Replace TwinStatus on DeviceTwin.

### CODE REVIEW

**Comment by mamokarz** [comment]
Thanks @timtay-microsoft

File: iot-e2e-tests/jvm/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothubservices/DeviceMethodAmqpsIT.java [hunk 1]
```diff
@@ -102,6 +99,19 @@ public void cleanToStart()
         x509Device.clearDevice();
     }
 
+    @After
+    public void delayTests()
+    {
+        try
+        {
+            Thread.sleep(2000);
+        }
```
Comment (prmathur-microsoft) [inline iot-e2e-tests/jvm/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothubservices/DeviceMethodAmqpsIT.java]: PSF please

Comment (prmathur-microsoft) [reply iot-e2e-tests/jvm/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothubservices/DeviceMethodAmqpsIT.java]: Here and everywhere...

Comment (mamokarz) [reply iot-e2e-tests/jvm/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothubservices/DeviceMethodAmqpsIT.java]: Sure

**Comment by mamokarz** [comment]
Thanks @prmathur-microsoft and @timtay-microsoft

**Pull request merged**